src/App.js

```diff
@@ -1,180 +1,221 @@
-import './App.css';
-import styled from 'styled-components'
-import React, { useReducer } from 'react'
-import { Activity, activityReducer } from './activity/Activity';
-import { Button } from './components/Theme'
+import "./App.css";
+import styled from "styled-components";
+import React, { useReducer } from "react";
+//import { Activity, activityReducer } from "./activity/Activity";
+import {
+  StickyNotesActivity,
+  activityReducer,
+} from "./StickyNotesActivity/StickyNotesActivity";
+import { Button } from "./components/Theme";
 
 const TabList = styled.div`
   border-bottom: 1px solid #ccc;
   padding-left: 0;
-`
+`;
 
 const sandboxReducer = (state, action) => {
   switch (action.type) {
-    case 'SELECT_TAB':
+    case "SELECT_TAB":
       return {
         ...state,
         [action.section]: {
-          activeTab: action.tabId
-        }
-      }
+          activeTab: action.tabId,
+        },
+      };
     case "TOGGLE_SPLIT":
       return {
         ...state,
-        split: !state.split
-      }
+        split: !state.split,
+      };
     default:
       return {
-        ...state
-      }
+        ...state,
+      };
   }
-
-}
+};
 
 const meetingReducer = (state, action) => {
   switch (action.type) {
     case "ADD_USER":
       return {
         ...state,
-
-      }
+      };
     case "REMOVE_USER":
       return {
         ...state,
-
-      }
+      };
     default:
       return {
         ...state,
         activities: {
-          "activity_instance": activityReducer(state.activities["activity_instance"], action)
-        }
-      }
+          activity_instance: activityReducer(
+            state.activities["activity_instance"],
+            action
+          ),
+        },
+      };
   }
-}
+};
 
 const initMeetingState = {
   activities: {
-    "activity_instance": {
-
-    }
+    activity_instance: {},
   },
   state: {
     users: {
       "alpha@aarvalabs.com": {
         userId: "alpha@aarvalabs.com",
         userName: "Alpha",
-        role: "host"
+        role: "host",
       },
       "bravo@aarvalabs.com": {
         userId: "bravo@aarvalabs.com",
         userName: "Bravo",
-        role: "guest"
-      }
-    }
-  }
-}
+        role: "guest",
+      },
```

Comment:
Is this some auto formatter that you've installed ?

Reply:
Yeah, it's called Prettier. It's a VSCode extension.

Reply:
I have been using prettier too. sometimes it does too much prettifying.